react/src/components/lablupTalkativotUI/LLMChatCard.tsx

```diff
@@ -163,6 +173,27 @@ const LLMChatCard: React.FC<LLMChatCardProps> = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [submitKey]);
 
+  useEffect(() => {
+    setTotalTokens(
+      _.sumBy(messages, (message) => message?.content.length) + input.length,
+    );
+  }, [messages, input]);
+
+  useEffect(() => {
+    if (!_.isEmpty(messages)) {
+      const lastMessage = _.last(messages);
+      if (lastMessage?.role === 'assistant') {
+        const lastToken = lastMessage.content?.length || 0;
+        if (startTime === null) {
+          setStartTime(Date.now());
+        } else {
+          const elapsedTime = (Date.now() - startTime) / 1000;
+          setTokenPerSecond(lastToken / elapsedTime);
+        }
+      }
+    }
+  }, [messages, startTime]);
```

Comment:
Run setStartTime before send request only .
```suggestion
useEffect(() => {
    if (!_.isEmpty(messages)) {
      const lastMessage = _.last(messages);
      if (lastMessage?.role === 'assistant' && startTime) {
        const lastToken = lastMessage.content?.length || 0;
        const elapsedTime = (Date.now() - startTime) / 1000;
        setTokenPerSecond(lastToken / elapsedTime);
      }
    }
  }, [messages, startTime]);
```